prototypes/ScheduledMerges.hs

```diff
@@ -416,6 +420,14 @@ update tr (LSMHandle scr lsmr) k op = do
       else
         writeSTRef lsmr (LSMContent wb' ls)
 
+supply :: LSM s -> Credit -> ST s ()
+supply (LSMHandle scr lsmr) credits = do
+    LSMContent _ ls <- readSTRef lsmr
+    modifySTRef' scr (+1)
+    supplyCredits credits ls
+    ok <- invariant ls
+    assert ok $ return ()
```

Comment:
I exposed this to make the test case simpler, just supplying merge credits instead of having to do inserts that are not relevant to the problem.